app/controllers/comments_controller.rb

```diff
@@ -1,19 +1,37 @@
 class CommentsController < ApplicationController
   before_action :cube_authentication
   def create
-    @comment = Comment.new(body: comment_body[:body], meetup_id: params[:meetup_id])
-    @comment.user = current_user
-    if @comment.save
-      redirect_to meetups_path
+    if request.xhr?
+      @comment = Comment.new(body: params[:body], meetup_id: params[:meetup_id])
+      @comment.user = current_user
+      meetupId = @comment.meetup.id
```

Comment:
Useless assignment to variable - `meetupId`.<br>Use snake_case for variable names.